tdrs-backend/tdpservice/users/migrations/0040_digit_team_permissions.py

```diff
@@ -0,0 +1,74 @@
+# Generated by Django 3.2.5 on 2021-08-16 14:10
+from django.contrib.auth.models import Group
+from django.db import migrations
+
+from tdpservice.users.permissions import (
+    get_permission_ids_for_model,
+    view_permissions_q
+)
+
+
+def add_digit_team_group(apps, schema_editor):
+    groups = [
+        Group(name="DIGIT Team"),
```

Comment:
Nit: understand we have other groups but suggest replacing space with "_" -> "DIGIT_Team"